ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15__okd.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
+  openshift_release_golang-1.19:
```

Comment:
Shouldn't this be `openshift_release_golang-1.20` given that is the input below?